code/game/machinery/doors/door.dm

```diff
@@ -19,7 +19,7 @@
 	var/heat_proof = 0 // For rglass-windowed airlocks and firedoors
 	var/emergency = 0 // Emergency access override
 	var/sub_door = 0 // 1 if it's meant to go under another door.
-	var/closingLayer = CLOSED_DOOR_LAYER
+	var/closingLayer =	 CLOSED_DOOR_LAYER

```

Comment:
I didn't touch that line at all(maybe I did and I didn't notice when I commited)